Clipper/joplin-webclipper/popup/src/bridge.js

```diff
@@ -41,15 +42,21 @@ class Bridge {
 				};
 
 				this.dispatch({ type: 'CLIPPED_CONTENT_SET', content: content });
+
+				if (command.useDefaultSettings) {
+					const state = this.store_.getState();
+					content.parent_id = state.selectedFolderId;
+					if (content.parent_id) {
+						this.sendContentToJoplin(content);
+					}
+				}
 			}
```

Comment:
I would prefer if each command has a well defined purpose. "clippedContent" is to set the clipped content within the extension, but shouldn't be used to also submit that content to Joplin directly. So is it possible to create a separate command, titled for example "sendContentToJoplin", which does the above? You might need to refactor the code a bit so as not to duplicate the content and response structures.